nw/guimain.py

```diff
@@ -466,14 +466,14 @@ def openProject(self, projFile):
                 lockDetails = (
                     "<br>%s" % self.tr(
                         "The project was locked by the computer "
-                        "'{computer_name}' ({os_name} {os_version}), "
-                        "last active on {time}"
+                        "'{computerName}' ({osName} {osVersion}), "
```

Comment:
I think this should be replaced by indexed placeholders to avoid overloading the translators. I did with named placeholders at the beginning of this work but changed my mind after stated translating the strings. If someone does a typo in the placeholder name, this can lead to runtime errors being thrown.

Reply:
Agreed. I thought you used named variables to make it easier for the translator to see the order. But yes, they should be numbers.